### PR TITLE
Add sample URLs for readonly mode to console output.

### PR DESCRIPTION
**Pull request by gokaysatir**
Change-Id: I2e10a25850a64bb666d0473d8881878ff41ad6b1

* Resolves: # 
* Target version: master 

### TODO

- [ ] ...

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

**Review by mmeeks — APPROVED** [review]
Looks great; do we propagate this to kit & core though ? and overwrite what we get from CheckFileInfo if readonly=true (but never if it is false =)

**Comment by gokaysatir** [comment]
> Looks great; do we propagate this to kit & core though ? and overwrite what we get from CheckFileInfo if readonly=true (but never if it is false =)

Yes we will be propagating the readonly flag to core side, but it is in another commit :) Waiting for approval.

* https://github.com/CollaboraOnline/online/pull/8488

**Pull request merged**
